### PR TITLE
Cleanup deadcode

### PR DESCRIPTION
**Pull request by Alfro**

### CODE REVIEW

**Comment by mreyk** [comment]
@Alfro This branch has conflicts that must be resolved and All checks have failed

File: src/Pumukit/ArcaBundle/DependencyInjection/PumukitArcaExtension.php [hunk 1]
```diff
@@ -19,9 +19,6 @@ class PumukitArcaExtension extends Extension
      */
     public function load(array $configs, ContainerBuilder $container)
     {
-        $configuration = new Configuration();
```
Comment (Alfro) [inline src/Pumukit/ArcaBundle/DependencyInjection/PumukitArcaExtension.php]: No configuration file in this bundle.

File: src/Pumukit/CoreBundle/Resources/views/Version/index.html.twig [hunk 2]
```diff
@@ -26,7 +26,7 @@
                 <tr>
                     <td>{{ item.type }}</td>
                     <td>{{ item.name }}</td>
-                    <td><span class="label label-success">{{ item.version }}</span></td>
+                    <td><span class="label label-success">{{ item.version }} - {{ item.dist.reference | slice(0,8) }}</span></td>
```
Comment (Alfro) [inline src/Pumukit/CoreBundle/Resources/views/Version/index.html.twig]: Not sure why I added this change here. It was probably a mistake when rebasing.

File: src/Pumukit/EncoderBundle/Resources/config/services.xml [hunk 3]
```diff
@@ -48,7 +48,6 @@
             <argument>%pumukitencode.thumbnail.width%</argument>
             <argument>%pumukitencode.thumbnail.height%</argument>
             <argument>%pumukit2.uploads_dir%/pic</argument>
-            <argument>%pumukit2.uploads_url%/pic</argument>
```
Comment (Alfro) [inline src/Pumukit/EncoderBundle/Resources/config/services.xml]: $targetUrl variable not used in the PicExtractorService

File: src/Pumukit/CoreBundle/Resources/views/Version/index.html.twig (hunk 2 above)
Comment (Alfro) [reply src/Pumukit/CoreBundle/Resources/views/Version/index.html.twig]: However, I really like this change (it shows the commit reference as well as the version, really useful to check installed branches on staging envs)

**Comment by Alfro** [comment]
Rebased and fixed checks. To review

**Comment by Yurujai** [comment]
Thanks!

**Pull request merged**